### PR TITLE
[Tesla] Fix change wrt Tesla’s back-end handling Event Stream errors

### PR DESCRIPTION
**Pull request by kgoderis**
Signed-off-by: Karel Goderis <karel.goderis@me.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java [hunk 1]
```diff
@@ -129,8 +131,10 @@
 
     protected boolean allowWakeUp = true;
     protected long lastTimeStamp;
-    protected long intervalTimestamp = 0;
-    protected int intervalErrors = 0;
+    protected long apiIntervalTimestamp = 0;
+    protected int apiIntervalErrors = 0;
+    protected long eventIntervalTimestamp = 0;
+    protected int eventIntervalErrors = 0;
```
Comment (wborn) [inline addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java]: Can you remove the redundant 0 initializations? They also generate more bytecode. I.e. slightly more storage and slightly less performance, see [Java Practices](http://www.javapractices.com/topic/TopicAction.do?Id=14).

Comment (kgoderis) [reply addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java]: @wborn done

**Comment by kgoderis** [comment]
@wborn can you merge? tx

**Comment by wborn** [comment]
Are the errors you are addressing with this PR due to a new/existing API rate limit @kgoderis?

**Comment by wborn** [comment]
Thanks for the fix and the info on why it occurs @kgoderis!
It doesn't seem like a very stable API if they continuously break the binding. :disappointed:

**Pull request merged**